spec/controllers/admin/signed_rebate_forms_controller_spec.rb

```diff
@@ -16,4 +16,27 @@
       expect(response.status).to eq(200)
     end
   end
+
+  describe 'GET download_csv' do
+    let!(:signed_forms) { FactoryBot.create_list(:signed_form, 3) }
+
+    before :each do
```

Comment:
🐘 `:each` is the default here so
```
before do 
  ...
end
```
would be equivalent. Just FYI in case you thought it was required. I kinda like this style of being explicit about it.